docs/content/routing/entrypoints.md

```diff
@@ -51,199 +56,414 @@ They define the port which will receive the requests (whether HTTP or TCP).
---entryPoints.EntryPoint0.proxyProtocol.insecure=true
---entryPoints.EntryPoint0.proxyProtocol.trustedIPs=foobar,foobar
---entryPoints.EntryPoint0.forwardedHeaders.insecure=true
---entryPoints.EntryPoint0.forwardedHeaders.trustedIPs=foobar,foobar
-```
-
-## ProxyProtocol
-
-Traefik supports [ProxyProtocol](https://www.haproxy.org/download/2.0/doc/proxy-protocol.txt) version 1 and 2.
+??? info "See the complete reference for the list of available options"
+    
+    ```toml tab="File (TOML)"
+    ## Static configuration
+    [entryPoints]
+      [entryPoints.name]
+        address = ":8888"
+        [entryPoints.name.transport]
+          [entryPoints.name.transport.lifeCycle]
+            requestAcceptGraceTimeout = 42
+            graceTimeOut = 42
+          [entryPoints.name.transport.respondingTimeouts]
+            readTimeout = 42
+            writeTimeout = 42
+            idleTimeout = 42
+        [entryPoints.name.proxyProtocol]
+          insecure = true
+          trustedIPs = ["127.0.0.1", "192.168.0.1"]
+        [entryPoints.name.forwardedHeaders]
+          insecure = true
+          trustedIPs = ["127.0.0.1", "192.168.0.1"]
+    ```
+    
+    ```yaml tab="File (YAML)"
+    ## Static configuration
+    entryPoints:
+      name:
+        address: ":8888"
+        transport:
+          lifeCycle:
+            requestAcceptGraceTimeout: 42
+            graceTimeOut: 42
+          respondingTimeouts:
+            readTimeout: 42
+            writeTimeout: 42
+            idleTimeout: 42
+        proxyProtocol:
+          insecure: true
+          trustedIPs:
+          - "127.0.0.1"
+          - "192.168.0.1"
+        forwardedHeaders:
+          insecure: true
+          trustedIPs:
+          - "127.0.0.1"
+          - "192.168.0.1"
+    ```
+    
+    ```bash tab="CLI"
+    ## Static configuration
+    --entryPoints.name.address=:8888
+    --entryPoints.name.transport.lifeCycle.requestAcceptGraceTimeout=42
+    --entryPoints.name.transport.lifeCycle.graceTimeOut=42
+    --entryPoints.name.transport.respondingTimeouts.readTimeout=42
+    --entryPoints.name.transport.respondingTimeouts.writeTimeout=42
+    --entryPoints.name.transport.respondingTimeouts.idleTimeout=42
+    --entryPoints.name.proxyProtocol.insecure=true
+    --entryPoints.name.proxyProtocol.trustedIPs="127.0.0.1,192.168.0.1"
+    --entryPoints.name.forwardedHeaders.insecure=true
+    --entryPoints.name.forwardedHeaders.trustedIPs="127.0.0.1,192.168.0.1"
+    ```
 
-If proxyprotocol header parsing is enabled for the entry point, this entry point can accept connections with or without proxyprotocol headers.
+### Forwarded Header
 
-If the proxyprotocol header is passed, then the version is determined automatically.
+You can configure Traefik to trust the forwarded headers information (`X-Forwarded-*`).
 
-??? example "Enabling Proxy Protocol with Trusted IPs" 
+??? info "`forwardedHeaders.trustedIPs`"
```

Comment:
```suggestion
??? info "`forwardedHeaders.trustedIPs`: Trusting Forwarded Headers from specific IPs."
```

Reply:
I prefer to be homogeneous with other element.
The name of the option can be long.